storage/heap/CMakeLists.txt

```diff
@@ -27,4 +27,4 @@ IF(WITH_UNIT_TESTS)
 
   ADD_EXECUTABLE(hp_test2 hp_test2.c)
   TARGET_LINK_LIBRARIES(hp_test2 heap mysys dbug strings)
-ENDIF()
\ No newline at end of file
+ENDIF()
```

Comment:
Here the change is just whitespace. You can remove the file from commit list I guess.